docs/ga-roadmap.md

```diff
@@ -0,0 +1,114 @@
+# Collector GA Roadmap
+
+This document defines the roadmap followed by the OpenTelemetry Collector, along with tentative dates and requirements for GA (stability).
+
+In this document, the term “OpenTelemetry Collector components” refers to all the golang modules and packages that are part of the “OpenTelemetry Collector” ecosystem which include the [core](https://github.com/open-telemetry/opentelemetry-collector) and [contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib) components. This term does not refer to the component concept inside the “OpenTelemetry Collector”.
```

Comment:
How about using a synonym, instead of using the term "component" for a different definition than the one we are used to? Suggestions would be: element, package, part, ...

Reply:
picked `packages`